root.tf

```diff
@@ -48,3 +55,132 @@ terraform {
     dynamodb_table = "tdr-terraform-github-state-lock"
   }
 }
+
+module "github_oidc_provider" {
```

Comment:
The migrated `modules` should be put in the `root_github_environment.tf` as they need to be created in each of the TDR environments (intg / staging / prod), eg each environment needs their own github actions role.

The `modules` need to have `count  = local.apply_environment` added to ensure they get added to each TDR environment, but not the `mgmt` account.

See the `README` about running the Terraform: https://github.com/nationalarchives/tdr-terraform-github/blob/main/README.md#running-the-project